#163411569 modify the modal

Chiazokam opened the pull request:
#### What does this PR do?
This PR modifies the appearance of the modal

#### Description of Task to be completed?
- reduce the width of the modal box
- close modal on clicking the screen

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-ui-view-modal-163411569 `
- open the index.html file on the browser and navigate to the desired pages

#### What are the relevant pivotal tracker stories?
[#163411569](https://www.pivotaltracker.com/story/show/163411569)

### Images
![image](https://user-images.githubusercontent.com/26940294/51585094-4de2f700-1ed8-11e9-8dc1-f6aa6d7ac87a.png)

Chiazokam committed:
feat (frontend): include column title in the view pages

- include column title in the view pages
- reduce the height of the party logo in mobile
- reposition the submit button on the vote page
- modify the hover color of text on the vote submit button
- make pages mobile responsive

[Delivers #163377772]

Chiazokam committed:
feat (frontend): include column title in the view pages

- include column title in the view pages
- reduce the height of the party logo in mobile
- reposition the submit button on the vote page
- modify the hover color of text on the vote submit button
- make pages mobile responsive
-  make the the row titles bold in the view pages
- remove texts in strip on view messages
- reduse the font size of the office column
- remove shadows in column titles
- make titles bold
- centralize vote submit button
- change column title on mobile view pages

[Delivers #163377772]

Chiazokam committed:
feat (frontend): make the the row titles bold in the view pages

-  make the the row titles bold in the view pages
- change the color of the title header and reduce height
- reduce spaces in the form
- make candidate and party display responsive
- link logo to the home page

[Delivers #163377772]

Chiazokam committed:
feat (frontend): make the the row titles bold in the view pages

-  make the the row titles bold in the view pages

[Delivers #163377772]

Chiazokam committed:
Merge branch 'ft-ui-column-title-163377772' of https://github.com/Chiazokam/Politico into ft-ui-column-title-163377772

Chiazokam committed:
Merge branch 'develop' into ft-ui-column-title-163377772

Chiazokam committed:
feat (frontend): modify the modal

- reduce the width of the modal box
- close modal on clicking the screen
- centralize modals

[Delivers #163411569]

Chiazokam committed:
feat (frontend): modify the modal

- reduce the width of the modal box
- close modal on clicking the screen
- centralize modals

[Delivers #163411569]

JCanaks commented:
@Chiazokam This is fine just make the reset password form centered and you will be good to go on this one

Chiazokam committed:
Merge branch 'ft-ui-column-title-163377772' of https://github.com/Chiazokam/Politico into ft-ui-column-title-163377772

Chiazokam committed:
feat (frontend): fix conflict on parent branch

	- include column title in the view pages
    - reduce the height of the party logo in mobile
    - reposition the submit button on the vote page
    - modify the hover color of text on the vote submit button
    - make pages mobile responsive
    -  make the the row titles bold in the view pages
    - remove texts in strip on view messages
    - reduse the font size of the office column
    - remove shadows in column titles
    - make titles bold
    - centralize vote submit button
    - change column title on mobile view pages

[Delivers #163440686]

Chiazokam committed:
feat (frontend): fix conflict on parent branch

	- include column title in the view pages
    - reduce the height of the party logo in mobile
    - reposition the submit button on the vote page
    - modify the hover color of text on the vote submit button
    - make pages mobile responsive
    -  make the the row titles bold in the view pages
    - remove texts in strip on view messages
    - reduse the font size of the office column
    - remove shadows in column titles
    - make titles bold
    - centralize vote submit button
    - change column title on mobile view pages

[Delivers #163440686]

Chiazokam committed:
Merge branch 'ft-ui-header-title-163440686' of https://github.com/Chiazokam/Politico into ft-ui-header-title-163440686

Chiazokam committed:
Merge pull request #23 from Chiazokam/ft-ui-header-title-163440686

#163440686 fix column title in the view pages

Chiazokam committed:
Merge branch 'ft-ui-view-modal-163411569' of https://github.com/Chiazokam/Politico into ft-ui-view-modal-163411569
